### DISCUSSION TITLE
Camera Image Not Aligning With Simulator Images

### DISCUSSION THREAD

**Discussion by jasoon-chen** [empriselab/RCareWorld#107] [Help - Programming] [answered]
I'm trying to get an RGB image from the camera. This is what I have:
```python
camera_hand = env.GetAttr(654321)
camera_hand.SetTransform(position=gripper.data['position'], rotation=[-90, 90, -90])
camera_hand.SetParent(3158930)
camera_hand.GetRGB(512, 512)
rgb = np.frombuffer(camera_hand.data["rgb"], dtype=np.uint8)
rgb = cv2.imdecode(rgb, cv2.IMREAD_COLOR)
cv2.imwrite("rgb.png", rgb)
```

This is what the simulator sees:
![image](https://github.com/user-attachments/assets/4579fe63-1b56-4edc-b0d0-bd1bd10a7a07)

This is what RGB.png sees:
![image](https://github.com/user-attachments/assets/cb3b47c1-c0bc-4617-abe7-8542291852cf)

It appears that the cloth is completely out of the RGB image but in the field of view of the camera. Is this a configuration issue with my camera?

**Comment by shxingch** [comment]
Hi @jasoon-chen.
The camera might be too close to the clothing, which could be causing the clipping issue. Could you try stepping back a bit and then take the picture again?

**Reply by jasoon-chen** [reply]
Sorry for the delayed response. I moved the camera back and I am experiencing the same issue. Is this reproducible on your end?
![image](https://github.com/user-attachments/assets/0927a068-143d-4f9b-a8b8-46ab5d0a79cd)
![image](https://github.com/user-attachments/assets/c0cf5574-e010-4161-b4ee-0e7631f131c2)

**Reply by shxingch** [reply]
Hi @jasoon-chen ! We have just updated the Dressing Player. Could you please update and try again?

**Reply by jasoon-chen** [reply]
I just updated it and now the dress is gone upon initializing of the environment. It appears the cloth just falls through the floor.
[Screencast from 08-26-2024 04:53:16 PM.webm](https://github.com/user-attachments/assets/277e7a04-6f7b-49f8-96c6-5283fb86251e)

**Reply by shxingch** [reply, marked as answer]
Hi @jasoon-chen , the camera bug has been fixed and the new Dressing Player has been updated!

**Reply by jasoon-chen** [reply]
Thanks! I'm now able to see the cloth. 
![image](https://github.com/user-attachments/assets/417fd4e7-7c9d-41a4-a616-fe10c4eac65a)
